scripts/west_commands/sign.py

```diff
@@ -444,6 +444,12 @@ def preprocess_toml(self, config_dir, toml_basename, subdir):
         # unfortunately don't seem to make any difference here and they're
         # gcc-specific)
         preproc_cmd += ['-P']
+        # REM allows standalone comments to be preserved in generated/*.toml.
+        # A "double" `REM # my comment...` is not necessary but recommended for
+        # syntax highlighting.
+        # Note trailing comments are ignored by cpp and don't need any REM
+        # trick, a trailing `#` is enough.
+        preproc_cmd += ['-DREM=#']
```

Comment:
would it work to define `REM` an empty string and require "REM #" in toml files?

Reply:
OK, will do.